hoplite-core/src/main/kotlin/com/sksamuel/hoplite/FileSource.kt

```diff
@@ -4,11 +4,13 @@ import com.sksamuel.hoplite.fp.Try
 import com.sksamuel.hoplite.fp.invalid
 import com.sksamuel.hoplite.fp.sequence
 import com.sksamuel.hoplite.fp.valid
+import java.io.File
+import java.io.FileInputStream
 import java.io.InputStream
 import java.nio.file.Files
 import java.nio.file.Path
 
-sealed class FileSource {
+abstract class FileSource {
```

Comment:
Maybe we should rename FileSource to ConfigSource since it's not always files.

Then FileIoSource can just be FileSource.

Reply:
Sounds good to me. Just tried to limit my changes as much as possible. :)